mergify_engine/tests/functional/test_subscription.py

```diff
@@ -25,14 +25,13 @@ async def test_subscription(self):
         self.subscription = subscription.Subscription(
             self.redis_cache,
             config.TESTING_ORGANIZATION_ID,
-            self.SUBSCRIPTION_ACTIVE,
             "Abuse",
             frozenset(
                 getattr(subscription.Features, f)
                 for f in subscription.Features.__members__
             )
             if self.SUBSCRIPTION_ACTIVE
-            else frozenset([]),
+            else frozenset([subscription.Features.PUBLIC_REPOSITORY]),
```

Comment:
This is the purpose of this test to check what happen when the feature flags list is empty.